menu changes according to the device controller

NelioQuibe-258 opened the pull request:
The following changes were made:
1-Modified Title Image
2.Added Script to the Scene Title.tscn
3. Added a generic label.

https://github.com/GamesForGood/CandyWrapperDeluxe/assets/79609481/98e5b5e1-8fd2-40bf-be8f-e6318927a81f

Filaipus commented:
Nelio, neste momento o menu muda se conectarmos um controlo a meio, mas perdeste a funcionalidade de começar com o menu que seria esperado. 

Ou seja, se eu já tiver o controle ligado ao meu computador o menu devia começar com "start" em vez de "space". Neste momento começa sempre com "space"

NelioQuibe-258 committed:
menu changes according to the device controller

The following changes were made:
1-Modified Title Image
2.Added Script to the Scene Title.tscn
3. Added a generic label.

NelioQuibe-258 committed:
menu changes according to the device controller

The following changes were made:
1-Modified Title Image
2.Added Script to the Scene Title.tscn
3. Added a generic label.

NelioQuibe-258 committed:
Merge branch 'change-menu-according-to-the-device-controller' of https://github.com/GamesForGood/CandyWrapperDeluxe into change-menu-according-to-the-device-controller

NelioQuibe-258 committed:
updated Title.dg as well as Title.tscn

By default, there's no any text on the general label.

When the game loads, it verifies which control is connected, afterwards it writes the text according to the device connected:
"space" for keyboards
"start" for joystick
